.github/docker/linux_amd64/Dockerfile

```
@@ -0,0 +1,124 @@
+FROM ubuntu:18.04
+
+###
+# Base image setup
+###
+
+ARG MAJOR=18
+
+# Setup the basic necessities
+RUN apt-get update -y -qq
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)